tests/testthat/test-models.R

```diff
@@ -0,0 +1,26 @@
+context("models")
+
+test_that("download models list", {
+  location <- montagu_test_server()
+  dat <- montagu_models_list(location = location)
+  expect_is(dat, "data.frame")
+  expect_equal(names(dat), c("id", "name", "citation", "modelling_group"))
+})
+
+test_that("download correct model id", {
+  location <- montagu_test_server()
+  dat <- montagu_model_by_id(model_id = "YFIC", location = location)
+  expect_is(dat, "list")
+  expect_equal(names(dat), 
+               c("id", "description", "citation", "modelling_group"))
+  expect_true(dat$id == "YFIC")
+})
+
+test_that("download incorrect model id", {
+  location <- montagu_test_server()
+  expect_error(montagu_model_by_id(model_id = "YFICZZZ", location = location), 
+    paste0("Error running request:\n\t - unexpected-error: ",
```

Comment:
Hmm, this is not a great error message to be working with!  Does this throw a 404?  that might be a stabler thing to check against here (you can do that by doing `tryCatch(montagu_model_by_id(model_id = "YFICZZZ", location = location), error = identity)` then working directly with the error object

Reply:
https://vimc.myjetbrains.com/youtrack/issue/VIMC-2550